pyerrors/input/json.py

```diff
@@ -479,8 +481,10 @@ def import_json_string(json_string, verbose=True, full_output=False):
     result : dict
         if full_output=True
     """
-
-    return _parse_json_dict(json.loads(json_string), verbose, full_output)
+    if json_string != "NONE":
```

Comment:
In the current version, this is always evaluated to be True, right?

Reply:
Yes, that is true, do you think we should return None or leave it as NaN, as it currently is?

Reply:
Actually, I don't really get why changes in `json.py` are necessary at all, as the routines work if the passed parameters have the correct form (A list of `Obs, list, numpy.ndarray, Corr` for `create_json_string` and a `json` string with data for `import_json_string`).  In my opinion it would be better to call the functions only if the parameters have the correct format and to handle the other cases where they appear (in this case: when applying the functions to the data frames) by not calling the functions. This would circumvent putting these hacks in the `json` functions.